crates/test_utils/wasm/wasm_workspace/migrate_new/src/coordinator.rs

```diff
@@ -18,7 +18,10 @@ fn init() -> ExternResult<InitCallbackResult> {
                 "Could not deserialize properties".to_string()
             ))
         })?;
-    open_chain(properties.prev_dna_hash.clone())?;
+
+    // TODO: must get close_hash from init context, which is currently not possible.
```

Comment:
This whole init() function will be rewritten because eventually the close_chain and open_chain hdk functions will go away, so the system can drive the creation of those actions.